code/modules/surgery/bodyparts/species_parts/moth_bodyparts.dm

```diff
@@ -1,3 +1,44 @@
+//SKYRAT EDIT START: New Moth code for moth code colorable body parts
```

Comment:
Put this in the modular folder where the other limbs are. Search _mutant_boryparts.dm